Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -66,6 +66,9 @@ import BraintreeCore
 
     /// Optional: Provides details to users about their recurring billing amount when using PayPal Checkout with Purchase.
     public var amountBreakdown: BTAmountBreakdown?
+    
+    /// Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`. Only available with PayPal Checkout.
+    public var shouldOfferCredit: Bool = false
```

Comment:
This can be added to the `BTPayPalRequest` and passed in the `super.init` for both Checkout and Vault requests.